python/plugins/processing/algs/qgis/ConcaveHull.py

```diff
@@ -69,13 +69,13 @@ def initAlgorithm(self, config=None):
         self.addParameter(QgsProcessingParameterBoolean(self.NO_MULTIGEOMETRY,
                                                         self.tr('Split multipart geometry into singleparts geometries'), defaultValue=False))
 
-        self.addParameter(QgsProcessingParameterFeatureSink(self.OUTPUT, self.tr('Concave hull'), type=QgsProcessing.TypeVectorPolygon))
+        self.addParameter(QgsProcessingParameterFeatureSink(self.OUTPUT, self.tr('Output layer'), type=QgsProcessing.TypeVectorPolygon))
```

Comment:
is it important this change?

Reply:
No, it was actually an accidental change (I wanted to change it in the k-nearest algorithm and changed it here by mistake). I will undo the change.